content/docs/studio/user-guide/projects-and-experiments/live-metrics.md

```diff
@@ -0,0 +1,102 @@
+repository. This requires a 3-step process:
+
+1. [Set up an access token](#set-up-an-access-token)
+2. [Configure your model training CI job](#configure-your-model-training-ci-job)
+3. [Send and view the updates](#send-the-updates)
+
+Live metrics must be sent to this ingestion end-point: TODO: Add the end-point
+here
+
+https://www.youtube.com/watch?v=hKf4twg832g
+
+TODO: Replace this with a video tutorial for live metrics
+
+## Set up an access token
+
+Iterative Studio uses access tokens to authorize [DVCLive] to send live updates
+to the metrics and plots.
+
+### Generate access token
+
+To set up the access token, open your user profile page. In the
+`Studio access token` section, click on `Generate new token`.
+
+### Regenerate access token
+
+You can regenerate the access token at any point. The old access token will no
+longer be authorized to send live metrics and plots to Iterative Studio.
+
+### Reset (delete) access token
+
+You can delete the access token when you no longer need it.
+
+The option to delete the access token is also available when you try to change
+your password. That is, you can reset all your access credentials (your password
+and the access token) at once. This is handy if you suspect that your account
+security may have been compromised.
+
+## Configure your model training CI job
+
+You should define the following environment variables in your CI job:
+
+- `STUDIO_REPO_URL`: URL for the Git repository connected to the project. E.g.,
+  `github:iterative/example-get-started`
+- `STUDIO_TOKEN`: Your access token. The access token must be present in any
+  request that sends data to the Iterative Studio ingestion endpoint. Requests
+  with missing or incorrect access tokens are rejected with an appropriate HTTP
+  error code and error message.
+
+One way to provide the above values to your CI job is to set them as GitHub
+secrets.
+
+Here is an example GitHub action. TODO: provide link to an example snippet (such
+as
+[this](https://github.com/iterative/test-dvclive-studio/blob/086a51d76c7983f24c091e1b007820916aa75e7d/.github/workflows/test_live_metrics.yaml#L17-L19))
+
+## Send the updates
+
+Now, whenever you log your metrics or plots using [DVCLive], they will be
+automatically sent to Iterative Studio.
+
+Iterative Studio stores the live Metrics in its database. In the project table,
+the live metrics are displayed next to the Git commit corresponding to the
+experiment.
+
+## View live metrics
+
+The live metrics are displayed alongside the corresponding experiment commits,
+and updates to the live metrics are highlighted in orange. The running
+experiments themselves are displayed with a `Running` indicator.
+
+TODO: Add a screenshot of the `Running` indicator
+
+The number of experiments with new updates to the live metrics values are
+displayed in the `Live` icon, which can also be used to filter and show only
+running experiments in the table.
+
+Live plots are displayed in the
```

Comment:
Minor: terminology consistency
```suggestion
The live metrics are plotted alongside the corresponding experiment commits,
and updates to the live metrics are highlighted in orange. The running
experiments themselves are displayed with a `Running` indicator.

TODO: Add a screenshot of the `Running` indicator

The number of experiments with new updates to the live metrics values are
displayed in the `Live` icon, which can also be used to filter and show only
running experiments in the table.

Plots of the live metrics are displayed in the
```

Reply:
> The live metrics are **plotted** 

Not really. The metrics scalars are displayed in the project table in the experiment (commit) row. They aren't really plotted.

> Plots of the live metrics

This is also not entirely true. The live plots are updates to the **plots** files, which may or may not have the same values as the live **metrics**. @mvshmakov pls correct me if I'm wrong.

Reply:
Oh OK. It's just unclear what "Live plots" are. I thought the history of live metrics was plotted, not that plots were updated in real time. These seem like different features but not well separated in [this page](https://dvc-org-live-metrics-bw8woadv9.herokuapp.com/doc/studio/user-guide/projects-and-experiments/live-metrics) so far. Probably adding image samples would help too.

Reply:
Got it. I've added `and plots` in all applicable places. Check out [this commit](https://github.com/iterative/dvc.org/pull/4012/commits/be466c550b19ef85ff638710ca849d9386fa22e3).

Reply:
For clarity, we may call such experiments (as a table row) "live experiments". These experiments may contain "live metrics" and "live plots". IIRC, we may theoretically have a situation when there are no live metrics, but only live plots set up and wise-versa as well.

So, what do you think about such terminology @tapadipti?